OpenRA.Game/Settings.cs

```diff
@@ -73,6 +73,9 @@ public class ServerSettings
 		[Desc("Disallow AI bots.")]
 		public bool LockBots = false;
 
+		[Desc("If a player drops, the AI should take over.")]
+		public bool BotsForDisconnectedClients = true;
```

Comment:
Is there a reason for a player to not want to have this? The original https://github.com/OpenRA/OpenRA/pull/8059 already had it in so it is easy to copy it, but it felt like a needless switch in the already crowded lobby.